website/source/docs/drivers/exec.html.md

```diff
@@ -30,6 +32,10 @@ The `exec` driver can run on all supported operating systems but to provide
 proper isolation the client must be run as root on non-Windows operating systems.
 Further, to support cgroups, `/sys/fs/cgroups/` must be mounted.
 
+You must specify a `command` to be executed. Optionally you can specify an
+`artifact_source` to be executed. Any `command` is assumed to be present on the 
+running client, or a downloaded artifact
```

Comment:
Can you put an example of how to run a downloaded artifact using $NOMAD_TASK_DIR for this and raw_exec.